distribution/docker/Dockerfile

```diff
@@ -90,6 +102,13 @@ RUN mkdir /opt/druid/var /opt/shared \
  && chown druid:druid /opt/druid/var /opt/shared \
  && chmod 775 /opt/druid/var /opt/shared
 
+# Install iproute2 to get the ip command needed to set config of druid.host IP address
+# Command needed in druid.sh Line 140;
+RUN if [ -x "$(command -v apt)" ] && [ ! -x "$(command -v ip)" ]; then \
```

Comment:
we probably don't need to check for apt here, since we assume either debian-based or distroless